test/AdoNetCore.AseClient.Tests/Integration/AseDataReaderTests.cs

```diff
@@ -643,6 +643,34 @@ public void GetByte_WithInvalidlyTypedValue_ThrowsException(string aseType, Type
             GetHelper_WithValue_ThrowsException(aseType, (reader, ordinal) => reader.GetByte(ordinal), exceptionType);
         }
 
+        [Test]
```

Comment:
Wrong PR for this test?

Reply:
Yup indeed. Not sure how that got in there :-)